pom.xml

```diff
@@ -48,6 +48,7 @@
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
 
+		<jbtis.version>4.5.0.AM1-SNAPSHOT</jbtis.version>
```

Comment:
The addition of this new .target file does not mean it's being USED. That will happen in a second PR. I just wanted you to test out the .target file and verify it works for you as expected, before we figure out how to use it inside your build.'